index.html

```diff
@@ -173,5 +263,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
```

Comment:
same as with title. If you want this wrapper over your title and subtitle, it's fine. But this wrapper will be the same in all sections. 
So `section-header` sounds a logic and meaningful class name for it. And you could use same class name in all sections